public/app/features/dashboard-scene/scene/NavToolbarActions.tsx

```diff
@@ -78,6 +79,11 @@ export function ToolbarActions({ dashboard }: Props) {
   const dashboardNewLayouts = config.featureToggles.dashboardNewLayouts;
   const isManaged = Boolean(dashboard.isManaged());
 
+  // Internal only;
+  // allows viewer editing without ability to save
+  // used for grafana play
```

Comment:
what is "grafana play"?

Reply:
@forsethc it's our play.grafana.org! it is a way to play around and see what grafana is about.

Reply:
Oh nice! makes sense that we'd want viewers to be able to edit stuff there.

If we're keeping this config flag around so play can use it, couldn't other configs also set it to true and get the same affect? (`NavToolbarActions.tsx` doesn't seem to be specific to play)